content/docs/gherkin/reference.md

```diff
@@ -260,6 +260,32 @@ Example: Multiple Givens
   But I shouldn't see something else
 ```
 
+### Star Notation
```

Comment:
This could be the star keyword.

Reply:
Yeah, I wasn't 100% either. It feels different to the actual keywords, being a sort of placeholder for any of them.

Reply:
@aslakhellesoy @sebrose do you have a view on what we call the asterisk thing?

Reply:
given that all the other words are called keywords and this operates as a keyword, keyword seems a good starting point perhaps?

Reply:
Yeah maybe I'm trying to be too clever in calling it something other than a keyword. In CSS, the `*` is called the "universal" selector which sort of fits I think. So maybe "Universal Keyword"

Reply:
@davidjgoss Is `*` just another way of spelling `And` or `But`? If so, why not just say that?

Reply:
It can also replace `Given`, `When`, `Then`. In fact, Cucumber ignores the keyword completely.

Reply:
Yeah given that is a placeholder for _any_ of the keywords then “universal” seems appropriate to me. I’ve updated but will leave this open a bit longer for more feedback

Reply:
I’ll admit that I don’t like “universal” in this sense; it seems that it’s trying to be too clever and not clear enough. That’s why I suggested just thinking of it as another way to spell `And`. 

It’s possible, though, that I don’t understand what star notation is meant for, so I may be misunderstanding its significance. (I wasn’t aware that it was in Gherkin before this issue brought it to my attention and, based on the examples given here, I can’t imagine ever using it...)

Reply:
Okay, so in the interest of getting this in - and address the gap in the docs - I've reworded to not refer to it by any name, but rather just call out the asterisk character and what it does.